src/commands/ingress/editTargetPort.ts

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzureWizard, AzureWizardPromptStep, IActionContext, nonNullProp, nonNullValueAndProp } from "@microsoft/vscode-azext-utils";
+import { ContainerAppItem } from "../../tree/ContainerAppItem";
+import { createSubscriptionContext } from "../../tree/ContainerAppsBranchDataProvider";
+import { IngressItem } from "../../tree/IngressItem";
+import { localize } from "../../utils/localize";
+import { pickContainerApp } from "../../utils/pickContainerApp";
+import { IContainerAppContext } from "../createContainerApp/IContainerAppContext";
+import { TargetPortStep } from "../createContainerApp/TargetPortStep";
+import { updateIngressSettings } from "./updateIngressSettings";
+
+export async function editTargetPort(context: IActionContext, node?: IngressItem): Promise<void> {
```

Comment:
I imagine a lot of this is just files that have been split up, renamed, etc. but aren't actually new code. Is it practical to try to denote which ones are actually new code--and should be reviewed closer?

Reply:
I did split up and rename a lot of the commands, however the implementations _did_ change too. So I would review those closely. 

Rewriting the tree items was a lot more of "copy this, copy that". So I'd say you can gloss over the tree item stuff. Potential mistakes there will be visible on the UI and should be easily caught by us or CTI.